### ISSUE TITLE
ember-cli-head doesn't get head template file with v3.4.2

### ISSUE THREAD

**Issue by josemarluedke** [ember-cli/ember-cli#8042]
The addon `ember-cli-head` relies on getting `app/templates/head.hbs` to inject the head into the application. It seems that with Ember CLI v3.4.2, the addon (`ember-cli-head`) doesn't get the `head.hbs` file as it injects a message in the html head saying the following:

```
 ember-cli-head/templates/head.hbs 
 If you see this your application's `head.hbs` has gone missing. 
```

I have putted together a reproduction of the bug here: https://github.com/josemarluedke/--test-cli-head

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.4.2
http_parser: 2.8.0
node: 10.7.0
v8: 6.7.288.49-node.15
uv: 1.22.0
zlib: 1.2.11
ares: 1.14.0
modules: 64
nghttp2: 1.32.0
napi: 3
openssl: 1.1.0h
icu: 62.1
unicode: 11.0
cldr: 33.1
tz: 2018e
os: darwin x64
6.1.0
1.9.4
```

**Comment by cah-briangantzler** [comment]
I use ember-page-title which depends on ember-cli-head to inject the title.

It is working correctly after upgrading to ember-cli 3.4.2

**Comment by BnitoBzh** [comment]
I encounter the same issue, it appeared in ember-cli@3.4.0 for me.
My workaround is to create a head-content component.

```
import Component from '@ember/component';
import { inject as service } from '@ember/service';

export default Component.extend({
  tagName: '',
  model: service('head-data')
});
```
And move your head.hbs content to the component template.

The issue seems to be that component can not use a template file outside templates/components folder.

@rwjblue what do you think about that?

**Comment by john-griffin** [comment]
Thanks, the `head-content` component work around fixes it in our app too.

**Comment by Turbo87** [comment]
this was resolved by #8048 and will be released soon

**Comment by rwjblue** [comment]
fix released in v3.4.3
